README.md

```diff
@@ -64,3 +56,51 @@ How to contribute ?
 - [Preethi G](https://github.com/preethical)
 - [Shivam Sharma](https://github.com/shivamragnar)
 - [Shreya Agrawal](https://github.com/shreyaagrawal0809)
+
+Folder structure
+----------------
+
+```
+.
+├── LICENSE
+├── README.md
+├── build
+│   ├── favicon.ico
+│   ├── logo192.png
+│   ├── logo512.png
+│   ├── manifest.json
+│   └── robots.txt
+├── package-lock.json
+├── package.json
+├── public
+│   ├── assets
+│   ├── favicon.png
+│   ├── index.html
+│   ├── logo192.png
+│   ├── logo512.png
+│   ├── manifest.json
+│   ├── robots.txt
+│   └── sitemap.xml
+├── src
+│   ├── App.js
+│   ├── App.scss
+│   ├── App.test.js
+│   ├── actions
+│   ├── components
+│   ├── content
+│   ├── data
+│   ├── dictionary
+│   ├── imgs
+│   ├── index.js
+│   ├── index.scss
+│   ├── logo.svg
+│   ├── reducers
+│   ├── scss
+│   ├── serviceWorker.js
+│   ├── store.js
+│   └── utils
+└── yarn.lock
+```
+License
+-------
+The code is licensed under MIT License while the contents inside the content folder are licensed under CC-BY-4.0.
```

Comment:
Link to CCBY page: https://creativecommons.org/licenses/by/4.0/

Reply:
Yo done